### ISSUE TITLE
Installer not working on Mac

### ISSUE THREAD

**Issue by bawee** [husonlab/dendroscope3#20]
Dear Dendroscope team,

The  installer (Dendroscope_macos_3_8_5.dmg) is unable to complete installation on my Macbook Pro with the apple M2 chip running Ventura 13.2.1. Is Dendroscope is compatible with the Apple Silicon macs? 

Thank you!
Bryan

**Comment by husonlab** [comment]
Could you please see whether 3_8_6 works for you... I have recreated the installer using an updated version of the installer-builder, which is supposed to fix some issues with Ventura. I will also ask someone in my group that has a Mac with the latest chip to see whether then can run the installer...

**Comment by bawee** [comment]
Yes it works now! thank you so much. It was horrifying to imagine life without Dendroscope! 

all the best,
Bryan
